model/user_access_token_test.go

```diff
@@ -16,43 +18,37 @@ func TestUserAccessTokenJson(t *testing.T) {
 	json := a1.ToJson()
 	ra1 := UserAccessTokenFromJson(strings.NewReader(json))
 
-	if a1.Token != ra1.Token {
-		t.Fatal("tokens didn't match")
-	}
+	require.Equal(t, a1.Token, ra1.Token, "tokens didn't match")
 
 	tokens := []*UserAccessToken{&a1}
 	json = UserAccessTokenListToJson(tokens)
 	tokens = UserAccessTokenListFromJson(strings.NewReader(json))
 
-	if tokens[0].Token != a1.Token {
-		t.Fatal("tokens didn't match")
-	}
+	require.Equal(t, tokens[0].Token, ra1.Token, "tokens didn't match")
 }
 
 func TestUserAccessTokenIsValid(t *testing.T) {
 	ad := UserAccessToken{}
 
-	if err := ad.IsValid(); err == nil || err.Id != "model.user_access_token.is_valid.id.app_error" {
-		t.Fatal(err)
-	}
+	err := ad.IsValid()
+	errCheck := err == nil || err.Id != "model.user_access_token.is_valid.id.app_error"
```

Comment:
no need for errCheck intermediary variable. just pass the condition to `require.False`

Reply:
I have made the changes.